toolbox

```diff
@@ -323,6 +323,15 @@ create_enter_command()
 
 create_environment_options()
 {
+    # have to call this outside of pipe to avoid "Inappropriate ioctl for device" error in stty.
+    terminal_size=$(stty size)
```

Comment:
Yes, this is definitely better than using `/dev/tty` because it will continue to work when `toolbox(1)` is used from inside a container and everything is forwarded to the host over D-Bus.

Nitpick: would be better to explain it in more detail in the commit message than a one-line comment. :)